extensions/infobip/v1/actions/sendSms/sendSms.ts

```diff
@@ -0,0 +1,67 @@
+import { z } from 'zod'
+import { type Action, Category, validate } from '@awell-health/extensions-core'
+import { type settings, SettingsValidationSchema } from '../../../settings'
+import { FieldsValidationSchema, fields } from './config'
+import { isEmpty } from 'lodash'
+import { infobipErrorToActivityEvent, isInfobipError } from '../../client/error'
+import { InfobipClient } from '../../client'
+
+export const sendSms: Action<typeof fields, typeof settings> = {
+  key: 'sendSms',
+  title: 'Send SMS',
+  description: 'Sends SMS using Infobip',
+  category: Category.COMMUNICATION,
+  fields,
+  previewable: false,
```

Comment:
What makes something previewable?

Reply:
Good question. The rule of thumb I am applying today is that any action that makes a call to a paid API (e.g. sending a text message will be charged in Infobip) is not previewable because we want to avoid customers being billed for previewing their care flows.

Looking forward, it should probably the user in Studio that should be able to decide whether they want actually test the care flow and all its actions - even if that means they'll be charged for some.